src/base/BaseScene.cc

```diff
@@ -755,6 +755,7 @@ CameraPtr BaseScene::CreateCamera(const std::string &_name)
 CameraPtr BaseScene::CreateCamera(unsigned int _id, const std::string &_name)
 {
   CameraPtr camera = this->CreateCameraImpl(_id, _name);
+  camera->SetUserData("user-camera", false);
```

Comment:
I think all cameras are created using this function, even camera sensors. I'd expect this to be something that is set after the camera is created by the caller of the function.